### PR TITLE
CI now bans nearspace that is not on turf/space

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

CI now bans nearspace that is not on turf/space (or other valid tiles like walls, and airless plating). Fixes the areas where this is broken as needed.

This PR also includes the fixes for it.

## Why It's Good For The Game
Area oversights are bad

## Testing

See CI, first commit will fail.

## Changelog
NPFC

### CODE REVIEW

File: code/modules/unit_tests/map_tests.dm [hunk 1]
```diff
@@ -109,3 +109,17 @@
 /datum/map_per_tile_test/structures_in_farspace_checker/CheckTile(turf/T)
 	if(T.loc.type == /area/space && locate(/obj/structure) in T.contents)
 		Fail(T, "tile contains at least one structure found in non-near space area")
+
+/datum/map_per_tile_test/nearspace_checker
+	var/allowed_turfs = list(/turf/space,
+							 /turf/simulated/floor/plating/airless,
+							 /turf/simulated/wall
+							)
```
Comment (AffectedArc07) [inline code/modules/unit_tests/map_tests.dm:117]: Space these to one indent after `allowed_turfs` not all the way.

File: code/modules/unit_tests/map_tests.dm [hunk 2]
```diff
@@ -112,9 +112,10 @@
 
 /datum/map_per_tile_test/nearspace_checker
 	var/allowed_turfs = list(/turf/space,
-							 /turf/simulated/floor/plating/airless,
-							 /turf/simulated/wall
-							)
+		/turf/simulated/floor/plating/airless,
+		/turf/simulated/floor/plasteel/airless,
+		/turf/simulated/wall
+		)
```
Comment (AffectedArc07) [inline code/modules/unit_tests/map_tests.dm:118]: This one level in.

Comment (Contrabang) [reply code/modules/unit_tests/map_tests.dm:118]: Like this? I suck at spacing out lists properly.
```
var/allowed_turfs = list(/turf/space,
		/turf/simulated/floor/plating/airless,
		/turf/simulated/floor/plasteel/airless,
		/turf/simulated/wall
	)
```

Comment (AffectedArc07) [reply code/modules/unit_tests/map_tests.dm]:
```
var/allowed_turfs = list(/turf/space,
	/turf/simulated/floor/plating/airless,
	/turf/simulated/floor/plasteel/airless,
	/turf/simulated/wall
)
```

Comment (AffectedArc07) [reply code/modules/unit_tests/map_tests.dm]: Yeah perfect

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**